tss-esapi-sys/build.rs

```diff
@@ -1,25 +1,140 @@
 // Copyright 2021 Contributors to the Parsec project.
 // SPDX-License-Identifier: Apache-2.0
 
-#[cfg(feature = "generate-bindings")]
+#[cfg(any(feature = "generate-bindings", feature = "bundled"))]
 use std::path::PathBuf;
 
+#[cfg(feature = "bundled")]
+use std::{
+    env,
+    path::Path,
+    process::Command,
+};
+
 const MINIMUM_VERSION: &str = "3.2.2";
 
+#[cfg(feature = "bundled")]
+fn fetch_source(dest_path: impl AsRef<Path>, name: &str, repo: &str, branch: &str) -> PathBuf {
+    let parent_path = dest_path.as_ref();
+    let repo_path = parent_path.join(name);
+    if !repo_path.join("Makefile.am").exists() {
+        let output = Command::new("git")
```

Comment:
I don't currently see a way of providing arguments to git clone in the git2 crate.

Reply:
Yeah, you might be right. It would have been nice. You would probably have to use the [RepoBuilder](https://docs.rs/git2/0.18.3/git2/build/struct.RepoBuilder.html) in order to add argument. But it is not that user friendly.